app/forms/draft_collection_form.rb

```diff
@@ -5,45 +5,48 @@
 class DraftCollectionForm < Reform::Form
   extend T::Sig
   feature EmbargoDate
+  include Composition
+  model 'collection'
```

Comment:
I know these were in the work version PR as well, but can you drop a comment here about what's going on?